### ISSUE TITLE
Bluetooth icon disapeared.

### ISSUE THREAD

**Issue by queeup** [elementary/wingpanel#224]
Issue #64 is solved and closed but I have still same problem. It disappears after waking from suspend. If I use `killall winganel` command, it fixes bluetooth icon.

![1](https://user-images.githubusercontent.com/87835/60045392-73656e00-96cd-11e9-99c7-f88750d864ea.png)
![2](https://user-images.githubusercontent.com/87835/60045443-909a3c80-96cd-11e9-9f80-c5439cee3b22.png)
```console
$ rfkill list
0: tpacpi_bluetooth_sw: Bluetooth
	Soft blocked: no
	Hard blocked: no
2: phy0: Wireless LAN
	Soft blocked: no
	Hard blocked: no
25: hci0: Bluetooth
	Soft blocked: no
	Hard blocked: no
```
```console
$ dpkg-query --showformat='${Package}\t${Version}\n' --show wingpanel wingpanel-indicator-bluetooth
wingpanel		        2.2.5+r269+pkg42~ubuntu5.0.1
wingpanel-indicator-bluetooth	2.1.3+r195+pkg25~ubuntu5.0.1
```

**Comment by peteruithoven** [comment]
I think this is an issue in the wingpanel code, at least with the previous issue it was. 

The reference the previous fix: https://github.com/elementary/wingpanel/pull/187

Last time I found out that it wouldn't happen when I commented the delayed re-ordering:
https://github.com/elementary/wingpanel/blob/1fbb38ced7ae3745d50ac10b67b6649f603313d2/src/Widgets/IndicatorEntry.vala#L83
But this means the indicator isn't removed and you'll see extra margins.

**Comment by queeup** [comment]
I am not having this problem atm. I update my system few days ago and everything look good. Closing.

**Comment by queeup** [comment]
I was wrong. I have still icon issue. I am not totally sure but I guess It happens while night light icon active.

**Comment by gayanper** [comment]
This happens to me with latest version of the ElementaryOS as well.

**Comment by KorbsStudio** [comment]
Yeah, this issue hasn't been fixed and as of 5.1.3 it still occurs.

**Comment by peteruithoven** [comment]
Please stop with the me 2 comments.

**Comment by wout** [comment]
I have something similar and probably related. Once or twice a month, Bluetooth stops working. Then when I restart it with `sudo systemctl restart bluetooth`, the icon disappears from the menu bar and never comes back. Also, all my devices are doubled or tripled in the dropdown menu:

![bluetooth](https://user-images.githubusercontent.com/107324/83385321-ea78ab80-a3e0-11ea-8409-fcd9612a5b69.png)

After running `killall winganel` everything is back to normal.

**Comment by bejaneps** [comment]
Having same issue, up !
